dvc/command/data_sync.py

```diff
@@ -22,9 +22,11 @@ def run(self):
 
 
 class CmdDataPull(CmdDataBase):
+    UP_TO_DATE_MSG = "Everything is up-to-date."
```

Comment:
Let's define this once in CmdDataBase above.

Reply:
https://github.com/iterative/dvc/pull/1620/commits/6898c465080338d6483d0ce9013fe50f4a0fb53b

I also noticed that `CmdDataStatus` has its UP_TO_DATE_MSG. I think it will override the message in `CmdDataBase`, right?